N/Netgen/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Netgen"
+version = v"1.5.199"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/RTimothyEdwards/netgen.git", "168e5502a1aeba4068beaa0437837ae748877ca8")
+]
+
+dependencies = [
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd netgen
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter!(p -> !Sys.iswindows(p), supported_platforms(;experimental=true))
```

Comment:
For the record
```suggestion
platforms = filter!(!Sys.iswindows, supported_platforms(;experimental=true))
```
works just fine :wink: